src/tools/rz-diff/binary_diffing.md

```diff
@@ -1,75 +1,219 @@
-# Binary Diffing
+# Rz-Diff (binary and text diffing utility)
 
-This section is based on the http://rizin.today article "[binary diffing](https://rizinorg.github.io/blog/posts/binary-diffing/)"
+[Diffing](https://en.wikipedia.org/wiki/Diff) is to show you the differences between two different versions of the same code. Rz-diff provides diffing several matrices Eg. distance, delta, percentage of similarity and even a hexadecimal diff.
```

Comment:
No need for explaining what diffing - everyone knows what is this. Also it isn't about matrices. I meant for particular algorithms.

Reply:
Oops! That was supposed to be metrics